resources/feature_dism.rb

```diff
@@ -25,7 +25,41 @@
 include Chef::Mixin::ShellOut
```

Comment:
I don't think shellout is needed to be included at this point, know it'snot a change here but just some cleanup maybe.